Source/WebCore/rendering/style/RenderStyleConstants.h

```diff
@@ -1092,6 +1099,13 @@ enum GridAutoFlow {
     AutoFlowColumnDense = InternalAutoFlowAlgorithmDense | InternalAutoFlowDirectionColumn
 };
 
+enum MasonryAutoFlow {
+    AutoFlowPack = InternalMasonryAutoFlowAlgorithmPack | InternalMasonryAutoFlowDefiniteFirst,
+    AutoFlowPackOrdered = InternalMasonryAutoFlowAlgorithmPack | InternalMasonryAutoFlowOrdered,
+    AutoFlowNext = InternalMasonryAutoFlowAlgorithmNext | InternalMasonryAutoFlowDefiniteFirst,
+    AutoFlowNextOrdered = InternalMasonryAutoFlowAlgorithmNext | InternalMasonryAutoFlowOrdered
```

Comment:
Please use OptionSets instead of bit math.

Reply:
Changed